ext/dep_gecode/dep_selector_to_gecode.cpp

```diff
@@ -100,9 +100,10 @@ void VersionProblemPool::DeleteAll()
 
 int VersionProblem::instance_counter = 0;
 
-VersionProblem::VersionProblem(int packageCount, bool dumpStats, bool debug, const char * logId)
+VersionProblem::VersionProblem(int packageCount, bool dumpStats, bool debug, const char * logId,
+    unsigned long int _timeout)
```

Comment:
Minor nitpick : I haven't noticed the use of _ (eg _timeout) in the codebase, may want to consider renaming to timeoutIn or something like that?